common/test/acceptance/pages/lms/courseware.py

```diff
@@ -63,6 +70,22 @@ def xblock_component_html_content(self, index=0):
         """
         return self.q(css=self.xblock_component_selector).attrs('innerHTML')[index].strip()
 
+    def xblock_components_contain_strings(self, expected_component_strings):
```

Comment:
Removed the method, added helper method with more helpful asserts in one of the test classes.